modules/lib/api.json

```diff
@@ -1787,11 +1787,11 @@
                 "summary": "Login the user",
                 "description": "Login the given user",
                 "tags": [ "user" ],
-                "operationId": "rutil:login",
+                "operationId": "auth:login",
                 "parameters": [
                     {
                         "name": "user",
-                        "in": "query",
+                        "in": "body",
```

Comment:
See https://github.com/eeditiones/roaster/blob/master/content/parameters.xql#L116. It appears to me that the current behavior of roaster will fall back on "query".

Reply:
body is definitely correct here.

Reply:
I guess my question is whether roaster supports "body"? From the link I provided, it seems "body" is not explicitly supported.